app/assets/stylesheets/cafe_works.css.scss

```diff
@@ -0,0 +1,19 @@
+.cafe-date {
+  background: $background_color;
+  border: 2px;
+  border-color: $border_color;
```

Comment:
Name of variable `border_color` should be written in all lowercase letters with hyphens instead of underscores